libs/pruviloj/Pruviloj/Core.idr

```diff
@@ -268,3 +274,37 @@ reflexivity =
                 , NamePart `{reflexivity}
                 , TextPart "is not applicable."
                 ]
+
+||| Attempt to swap the sides of equality in a goal.
+symmetry : Elab ()
+symmetry = do
+    [_,_,_,_,_] <- apply (Var `{{sym}}) [True, True, True, True, False]
```

Comment:
The handling of holes here is too implicit. The code would be more clear if it explicitly bound the name of the last hole (the one that will have an equality proof in it later) and then explicitly called `focus` on that, and the docstring stated that if the original goal is `x = y`, then after this tactic, the focus will be on a hole of type `y = x`.

Alternatively, it could return the name of the hole and rely on the caller to focus on it, but I think that the caller will almost always want to proceed in the hole, so the version that calls `focus` is probably more convenient.